requirements.txt

```diff
@@ -12,7 +12,7 @@ pandas>=1.1.3,<2.2; python_version == "3.9"
 pandas>=1.3.0,<2.2; python_version >= "3.10"
 pandas<2.2; python_version >= "3.12"
 # analytics
-posthog>3,<6
```

Comment:
Can we keep the lower pin and leave this a range? If someone is importing GX as a dependency in their app, we don't want to force them to use a particular version of posthog.